system/scripts/npc/city/c_Klaipe.lua

```diff
@@ -0,0 +1,40 @@
+addnpc(20115,"ETC_20150717_013603","c_Klaipe",510.7029,-1.292879,-349.3194,45,"DummyCall")
```

Comment:
Could you insert spaces between the parameters, so it's easier to read, with all those numbers, dots, and commas? It will be relatively easy for you, I'd have to go into each file and format it^^

E.g.
```lua
addnpc(20115, "ETC_20150717_013603", "c_Klaipe", 510.7029, -1.292879, -349.3194, 45, "DummyCall")
```